src/components/Book.css

```diff
@@ -0,0 +1,176 @@
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 30px;
+  height: 160px;
+  margin-bottom: 20px;
+}
+
+.book-container:last-child {
+  margin: 0;
+}
+
+.book-details {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.category {
+  opacity: 0.5;
+  font-family: sans-serif, Montserrat;
+  font-size: 14px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
You have these properties repeated in several rules. It is better to put such properties in their own class to avoid repetition. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

Reply:
Done Thanks